susemanager/src/mgr_bootstrap_data.py

```diff
@@ -1098,6 +1098,10 @@
         'BASECHANNEL' : 'opensuse_leap15_1-x86_64', 'PKGLIST' : PKGLIST15_TRAD + PKGLIST15_SALT + PKGLIST15SP0SP1_SALT + PKGLIST15_X86_ARM,
         'DEST' : '/srv/www/htdocs/pub/repositories/opensuse/15/1/bootstrap/'
     },
+    'openSUSE-Leap-15.1-aarch64-uyuni' : {
+        'BASECHANNEL' : 'opensuse_leap15_1-aarch64', 'PKGLIST' : PKGLIST15_TRAD + PKGLIST15_SALT + PKGLIST15SP0SP1_SALT + PKGLIST15_X86_ARM,
+        'DEST' : '/srv/www/htdocs/pub/repositories/opensuse/15/1/bootstrap/'
```

Comment:
Since we do the same (using the same `DEST` for all SLE architectures), I guess this should be fine. However I wonder we shouldn't document at the doc something. As if someone recreates the repo with `--flush` the other architectures will be gone.

Reply:
Hmm, this reminds me of a problem with the automatic re-creation of bootstrap repos.

Reply:
Now I see the other PRs, but still... should we document this?

Even if automatic bootstrap repo generation works for the `*-uyuni`channels (does it?), there's still the risk of having the user generating them.

I understand that if you have the bootstrap repo created and if you *do not use* `--flush`, the packages will be added, and things should work, right?